controllers/attractor/lb-fe.go

```diff
@@ -129,13 +127,18 @@ func (l *LoadBalancer) getFeEnvVars(allEnv []corev1.EnvVar) []corev1.EnvVar {
 			Name:  "NFE_NAMESPACE",
 			Value: l.attractor.ObjectMeta.Namespace,
 		},
+		{
+			Name:  "NFE_EXTERNAL_INTERFACE",
+			Value: common.GetExternalInterfaceName(l.attractor),
+		},
 	}
 
 	for _, e := range allEnv {
 		// append all hard coded envVars
 		if e.Name == "SPIFFE_ENDPOINT_SOCKET" ||
 			e.Name == "NFE_LOG_BIRD" ||
-			e.Name == "NFE_ECMP" {
+			e.Name == "NFE_ECMP" ||
+			e.Name == "NFE_LOG_LEVEL" {
```

Comment:
I don't see NFE_LOG_LEVEL in Meridio nsm uplift PR. Shouldn't it exist in both PRs?